src/Blockcore.Dns/Dockerfile.Release

```diff
@@ -0,0 +1,13 @@
+FROM mcr.microsoft.com/dotnet/core/aspnet:3.1-buster-slim
+WORKDIR /usr/local/app/
+
+RUN apt-get update \
+    && apt-get install -y curl \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN curl -Ls https://github.com/block-core/blockcore-dns/releases/download/#{VERSION}#/Blockcore.dns-#{VERSION}#-linux-x64.tar.gz \
```

Comment:
"Blockcore.dns", is that the same as output from build? Should be correct case at least.

Reply:
I sort of just did similar to the indexer project